src/ixdat/quant_patch.py

```diff
@@ -0,0 +1,99 @@
+"""Patches corresponding to code that should be added to Spectro Inlets quantification"""
+
+
+def append_sensitivity_factors(*args):
+    """Recursively append elements in a list"""
+    sf = append_two_sensitivity_factors(args[0], args[1])
+    if len(args) == 2:
+        return sf
+    remaining_args = [sf] + list(args[2:])
+    return append_sensitivity_factors(*remaining_args)
+
+
+def append_two_sensitivity_factors(sf_1, sf_2):
+    """This should be incorporated into __add__ of SensitivityFactor and SensitivityList
+
+    Appends all the sensitivity factors in sf_1 and sf_2, which can each be a single
+    SensitivityFactor (or CalPoint) or a SensitivityList (or Calibration).
+
+    Args:
+        sf_1 (SensitivityFactor or SensitivityList): The first sensitivity factor or list
+            thereof.
+        sf_2 (CalPoint or Calibration): The second sensitivity factor or list thereof
+    """
+    from spectro_inlets_quantification.sensitivity import (
```

Comment:
Yeah, that was the idea. I think it has to be optional so long as it's not in PyPI...

Reply:
So we already sorted out a way to do the imports nicely, but I think the question remains, even with siq in PyPI, whether we make plugins mandatory dependencies or not. As I argues elsewhere, I think since disks are cheap, the argument against is small, but still perhaps something to consider.